src/utils/focusVisible.js

```diff
@@ -0,0 +1,149 @@
+/*
+Copyright (c) 2018-2020 Uber Technologies, Inc.
+
+This source code is licensed under the MIT license found in the
+LICENSE file in the root directory of this source tree.
+*/
+
+// @flow
+/* global window */
```

Comment:
We should wrap `window` usages in the `if (__BROWSER__) {` instead. Those will be needed on a client-side only right?

Reply:
functions using `window` are called only from componentDidMount or onFocus / onBlur. What would `if (__BROWSER__) {` achieve? If anything it's probably better to see server errors if we use those incorrectly.